### ISSUE TITLE
Is it possible to include the style in the html?

### ISSUE THREAD

**Issue by KieronWiltshire** [artf/grapesjs#3383]
I'm currently using grapesjs to build email templates, however when I import HTML with the <style> tag already inside, before this is stored it split's the HTML and CSS up. When this is stored in my backend database, there is no <style> or <link> tag in the HTML, instead it's stored separately. Is there a way to prevent this and store a single compiled version of both, or at least is there a way to inject a style with the CSS before sending it to the server?

**Comment by Ju99ernaut** [comment]
Grapesjs doesn't support inlinecss, but you can use the same technique used here https://github.com/artf/grapesjs-preset-newsletter. It uses the `juice` package to merge the html and css

**Comment by artf** [comment]
Yeah, at the moment the inline CSS is parsed for the CssComposer and [removed from the DOM](https://github.com/artf/grapesjs/blob/bb24e02c66c6bcb3104bf2c99fd3ef30a4d1e1f3/src/parser/model/ParserHtml.js#L290-L302). This allows the editor to edit its styles.

**Comment by ronaldohoch** [comment]
One question about it. 

How can ckeditor edit putting the span with inline styles and keep it after close and reopen the same template?
